### PR TITLE
[SDTEST-60] add information on total code coverage support in the Ruby test visibility library

### PR DESCRIPTION
**Pull request by anmarchenko**
### What does this PR do? What is the motivation?
Adds info on a newly released total code coverage support for Ruby, updates test visibility libraries features table.

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

**Review by juan-fernandez — APPROVED** [review]
🚀

**Review by jhgilbert — APPROVED** [review]

**Comment by jhgilbert** [comment]
Hi @anmarchenko, I've approved this on behalf of the docs team, but I see there is one more reviewer. Would you like me to wait for their approval before merging? Thanks!

**Comment by anmarchenko** [comment]
@jhgilbert no, from the @DataDog/ci-app-libraries team one review is enough, approval was given by @juan-fernandez

**Review by romainkomorndatadog — APPROVED** [review]

**Pull request merged**
